### PR TITLE
Fixes #31440 - adding snippet_if_exists to nochroot block

### PR DESCRIPTION
**Pull request by domitea**
Adding call of macro snippet_if_exists for better logging in nochroot block of kickstart_default template.

### CODE REVIEW

**Comment by lzap** [comment]
Nice, can you add this after TTY change (the next two lines) so user actually see what's going on?

Also you need to regenerate snapshot template test fixtures after the change.

    RAILS_ENV=test bundle exec rake snapshots:generate

**Comment by domitea** [comment]
@lzap Added line moved two lines below (after TTY change) and fixtures regenerated. Looks good for you?

File: app/views/unattended/provisioning_templates/provision/kickstart_default.erb [hunk 1]
```diff
@@ -246,6 +246,7 @@ redhat-lsb-core
 %post --nochroot --log=/mnt/sysimage/root/install.postnochroot.log
 /usr/bin/chvt 3
 echo "Changed to TTY3 for post installation..."
+<%= snippet_if_exists(template_name + " custom postnochroot") %>
```
Comment (ares) [inline app/views/unattended/provisioning_templates/provision/kickstart_default.erb]: if you add `-` to `%>` it won't even affect the snapshot, it wouldn't be adding extra new line

**Review by ares — CHANGES_REQUESTED** [review]
one tiny thing, then good to merge I believe

**Comment by domitea** [comment]
@ares PR updated

**Review by ares — APPROVED** [review]
Thanks @domitea, merging

**Comment by lzap** [comment]
Sorry for the delay, I just started with post-break GH backlog.

**Pull request merged**